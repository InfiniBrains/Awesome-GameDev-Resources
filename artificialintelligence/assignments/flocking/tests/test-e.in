0.000 2.000 1.000 1.000 0.000 2.000 3.000 2
0.000 0.500 -2.000 -1.000
0.000 -0.500 2.000 3.000
0.125
