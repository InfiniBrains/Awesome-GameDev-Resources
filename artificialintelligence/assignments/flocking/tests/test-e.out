-0.250 0.453 -2.000 -0.375
0.250 -0.109 2.000 3.125
